core/src/main/java/com/graphhopper/reader/dem/USGSProvider.java

```diff
@@ -12,7 +12,7 @@
 import org.apache.xmlgraphics.image.codec.util.SeekableStream;
 
 public class USGSProvider extends AbstractTiffElevationProvider {
-
```

Comment:
Remove

Reply:
Remove the `filename` variable too, your `getFileName()` method generalizes which tile we will use for returning elevation in `getEle()`.